release/nightly_tests/chaos_test.yaml

```diff
@@ -10,7 +10,7 @@
 
   run:
     timeout: 3600
-    prepare: python wait_cluster.py 10 600
+    prepare: python wait_cluster.py 10 600; python setup_chaos.py --no-start
```

Comment:
--no-start here because each test compares the performance between chaos testing on vs off. The application decides when to start chaos testing in this way.

Reply:
Actually, can we remove the --no-start option? I think specifying a timeout is probably enough. It would be nice to have the fixture set up so that you don't have to modify the normal test at all. So that way we can just copy all of the existing test yamls and add the fixture.

Can you also try applying this to an existing test? Then I think we could also remove the new tests in chaos_test (since I think they pretty much duplicate existing tests anyway).

Reply:
I don't know what tests can have the same verification as this? For example, I don't know if we have a test that can check at least once feature of actor restart. For tasks, maybe we can add it to many_tasks, but imo, the scale is too big, and the test itself is not that stable to introduce chaos test. This is same for placement group.   I am open to apply fixture to existing tests, but lmk which tests you'd like to try this.

> Actually, can we remove the --no-start option?

I think the complexity of this flag is pretty low, and you are free to not use it? I think having the ability to control this could be very useful when we debug issues in the future.

Reply:
Regarding no-start. My opinion is not strong. If you think it is not that useful, I can remove it.

Reply:
Hmm I guess if we can't transparently report the number of nodes killed anyway, then it's okay.

But we should try to improve the fixture so that it's completely transparent (no --no-start flag or manual node reporting) in the future. I'm not worried about complexity, just about making the fixture foolproof to use for future tests.

Also, we can merge this as is, but we really should try to apply the fixture to existing tests. Otherwise, we can't be sure of test coverage. Here I'm trying to guard against our just making new tests pass vs. finding FT problems with existing tests. If the scale is too big, we can reduce the kill interval accordingly.

Reply:
> But we should try to improve the fixture so that it's completely transparent (no --no-start flag or manual node reporting) in the future. I'm not worried about complexity, just about making the fixture foolproof to use for future tests.

I totally agree with this. Let me make a follow up to improve this to our lineage reconstruction roadmap. 

> but we really should try to apply the fixture to existing tests.

This also sounds good. Let's discuss what tests to add in the next sync up.